src/sentry/static/sentry/app/views/organizationDiscover/queryBuilder.jsx

```diff
@@ -48,6 +49,11 @@ export default function createQueryBuilder(initial = {}, organization) {
   const defaultProjects = organization.projects
     .filter(projects => projects.isMember)
     .map(project => parseInt(project.id, 10));
+
+  const projectsToFetchTags = ConfigStore.get('user').isSuperuser
+    ? organization.projects
```

Comment:
This adds another place we're relying on all the data for an account being present in the client. Will the API do the right thing if we omit the project list and send no value?

Reply:
No, Discover always needs a list of projects. Since we're still figuring out where this data should come from let's leave it this way for now (it doesn't need detailed project objects and it's only passed into the querybuilder once and used here)

Reply:
Ok :+1: